### ISSUE TITLE
ContraD results

### ISSUE THREAD

**Issue by STomoya** [STomoya/animeface#67]
Could not get good results in ContraD.

## what can be done.
- Train with more simpler model. (vs StyleGAN2)
- Further hyper-parameter tuning.
- Loss function. (taken from original implementation so I believe it's not wrong.)
